src/Utils/filterUtils.js

```diff
@@ -0,0 +1,51 @@
+const getSortedProducts = (products,sortBy)=>{
```

Comment:
You can add jsdoc documentation for better understanding of code

Reply:
ok, will include that.